### ISSUE TITLE
Better pause screen options menu

### ISSUE THREAD

**Issue by axollyon** [HackerN64/HackerSM64#562]
Create a vanilla-like menu in the pause screen to replace the prompt to press L to switch between widescreen modes.
Potential options to include:
- Aspect ratio
- Sound mode
- Game volumes (I have an implementation of this that I've used for my hacks where you can control the volume of sound effects and background music separately)
- Language (if MULTILANG is enabled)
Do not start on this until #560 (ASCII/UTF-8 implementation) is merged.

**Comment by thecozies** [comment]
volume control is pretty devisive. SFX is an interesting option though. sound mode is a weird one, we need to remove the headset option so we can get better stereo imaging @gheskett, but then the other option is mono lmao
I'm not really sure what else could be added to this menu outside of game specific things (or camera)

**Comment by Reonu** [comment]
mono is good for accesibility, there are people who can't hear in one ear and stuff like that. then again if you're playing on emu you can force mono at an OS level with the accessibility settings there idk

**Comment by thecozies** [comment]
yeah might as well keep it if @gheskett agrees it's not too bad to have around

**Comment by FazanaJ** [comment]
nerds playing on bad tv's only have mono output, plus mono is less CPU usage, so there's definitely merit to keeping mono

**Comment by gheskett** [comment]
> yeah might as well keep it if @gheskett agrees it's not too bad to have around

As much as it pains me internally, I would agree that mono needs to stay as an option by default.

**Comment by gheskett** [comment]
The thing we _don't_ care about is headset, though particularly once I go through and rip out stereo headset effects

**Comment by axollyon** [comment]
imo if we want to change what sound modes there are, that might be out of scope for this issue, this is just to make that configurable from places other than file select
if we're changing what sound modes we have, that should be a separate pr

**Comment by gheskett** [comment]
Yeah sound modes should be a separate PR, but it shouldn't be much more than an hour of work to implement properly. Maybe just wait for that to be ready before worrying about this portion I guess.

**Comment by Reonu** [comment]
new option idea: dither filter

could be on/off/auto (auto would automatically disable it when RDP usage gets close to 100% and enable it again once it stays below a set threshold for a certain amount of time, kinda how like dynamic resoluiton works in modern games)

if the auto thing is too complicated then a simple on/off is good too but I think auto would be cool

**Comment by axollyon** [comment]
> new option idea: dither filter
> 
> could be on/off/auto (auto would automatically disable it when RDP usage gets close to 100% and enable it again once it stays below a set threshold for a certain amount of time, kinda how like dynamic resolution works in modern console games)
> 
> if the auto thing is too complicated then a simple on/off is good too but I think auto would be cool

somebody else should be the one to develop that, but i'll take it if only to make the options menu less empty (especially if we don't do volume sliders by default)

**Comment by Reonu** [comment]
I mean we could add a simple on/off toggle for now and just have the auto thing noted as a possible future feature
